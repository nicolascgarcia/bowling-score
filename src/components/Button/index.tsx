import styled from 'styled-components';
import {
	layout,
	LayoutProps,
	typography,
	TypographyProps,
} from 'styled-system';

type ButtonProps = TypographyProps & LayoutProps;

export default styled.button<ButtonProps>`
	min-width: 50px;
	height: 50px;
	background-color: #8257e5;
	border-radius: 10px;
	display: flex;
	justify-content: center;
	align-items: center;
	border: none;
	color: #fff;
	font-weight: bold;
	font-size: 22px;
	text-transform: uppercase;
	${layout}
	${typography}

  :hover {
		background-color: #6946b9;
	}
`;
