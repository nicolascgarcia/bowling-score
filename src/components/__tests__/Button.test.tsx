/* eslint-disable import/no-extraneous-dependencies */
import renderer from 'react-test-renderer';
import Button from '../Button';

describe('<Button/>', () => {
	it('should render Button', () => {
		const component = renderer.create(<Button>Jest</Button>);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
