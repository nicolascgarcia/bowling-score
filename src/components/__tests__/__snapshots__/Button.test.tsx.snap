// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> should render Button 1`] = `
<button
  className="sc-bczRLJ idcXHr"
>
  Jest
</button>
`;
